rust_src/src/multibyte.rs

```diff
@@ -87,6 +87,15 @@ impl LispStringRef {
         s.size
     }
 
+    /// Return width of STRING when displayed in the current buffer. Width is
+    /// measured by how many columns it occupies on the screen. When calculating
+    /// width of a multibyte character in STRING, only the base leading-code is
+    /// considered; the validity of the following bytes is not checked.  Tabs in
+    /// STRING are always taken to occupy `tab-width' columns.
+    pub fn string_width(self) -> ptrdiff_t {
```

Comment:
```suggestion
    pub fn width(self) -> usize {
```

We can avoid "stuttering". The lisp name is `<type>-<field>` and directly analogous to object oriented `<type>::<field>`.

Also, we are using `usize` instead of `ptrdiff_t`. Same thing, different name.

Reply:
Oh right, I hadn't realized it was following that convention. I remember GTK's `gtk_text_tag_table_new()` style names, really makes you appreciate a proper module system.